plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -61,7 +61,7 @@ func NewSelectorSpreadPriority(
 
 // Returns selectors of services, RCs and RSs matching the given pod.
 func getSelectors(pod *v1.Pod, sl algorithm.ServiceLister, cl algorithm.ControllerLister, rsl algorithm.ReplicaSetLister, ssl algorithm.StatefulSetLister) []labels.Selector {
-	selectors := make([]labels.Selector, 0, 3)
```

Comment:
quick question here : why was 3 chosen originally , and is making the slice potentially > 3 possibly going to lead to reduced test coverage?  i assume `no` but just double checking :).